<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>save</class>
 <widget class="QWidget" name="save">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>325</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>613</width>
    <height>325</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>613</width>
    <height>325</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>11</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Save</string>
  </property>
  <property name="windowIcon">
   <iconset resource="maya.qrc">
    <normaloff>:/shelf/img/shelf/shelf_save35.png</normaloff>:/shelf/img/shelf/shelf_save35.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(31, 31, 31);</string>
  </property>
  <widget class="QLabel" name="lblImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>416</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="maya.qrc">:/shot/img/shot/000.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxPublish">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>197</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>PUBLISH</string>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtSavePath">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>195</y>
     <width>361</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>140_LIGHT_v003_ar.ma</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFolder">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>196</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/folderBtn24.png</normaloff>:/btn/img/btn/folderBtn24.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>231</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/btnSave.png</normaloff>:/btn/img/btn/btnSave.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>271</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/denialBtn24.png</normaloff>:/btn/img/btn/denialBtn24.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>23</width>
     <height>23</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="edtComment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>231</y>
     <width>501</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
   <property name="centerOnScroll">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="edtMetaData">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>41</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>Resolution:	2048 x 1152

FPS:	25 FPS

Frames:	1001 - </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHelp">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>301</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/helpBtn24.png</normaloff>:/btn/img/btn/helpBtn24.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReport">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>301</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/mailBtn24.png</normaloff>:/btn/img/btn/mailBtn24.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEmail">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>301</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="maya.qrc">
     <normaloff>:/btn/img/btn/inboxBtn24.png</normaloff>:/btn/img/btn/inboxBtn24.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>176</y>
     <width>596</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblShotNr">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>151</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent';</string>
   </property>
   <property name="text">
    <string>140</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblBanner">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>11</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent';</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="maya.qrc">:/banner/img/banner/LIGHT.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblVersion">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>161</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent';</string>
   </property>
   <property name="text">
    <string>v.0.01</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxUser">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblUser">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>6</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: 'transparent';</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="maya.qrc">:/user/img/user/_default.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>193</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>WORK</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblSlash">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>193</y>
     <width>10</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>|</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVersionUp">
   <property name="geometry">
    <rect>
     <x>455</x>
     <y>196</y>
     <width>21</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string>↑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVersionDown">
   <property name="geometry">
    <rect>
     <x>455</x>
     <y>208</y>
     <width>21</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Malgun Gothic</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(56, 56, 56);</string>
   </property>
   <property name="text">
    <string>↓</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtPath">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>501</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>P:/2_production/2_shots/140_magneto/4_LIGHT/WORK/140_LIGHT_v003_ar.ma</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>cbxUser</zorder>
  <zorder>line</zorder>
  <zorder>edtComment</zorder>
  <zorder>lblImage</zorder>
  <zorder>cbxPublish</zorder>
  <zorder>edtSavePath</zorder>
  <zorder>btnFolder</zorder>
  <zorder>btnSave</zorder>
  <zorder>btnCancel</zorder>
  <zorder>edtMetaData</zorder>
  <zorder>lblShotNr</zorder>
  <zorder>lblBanner</zorder>
  <zorder>lblVersion</zorder>
  <zorder>btnEmail</zorder>
  <zorder>btnHelp</zorder>
  <zorder>btnReport</zorder>
  <zorder>lblUser</zorder>
  <zorder>lblStatus</zorder>
  <zorder>lblSlash</zorder>
  <zorder>btnVersionUp</zorder>
  <zorder>btnVersionDown</zorder>
  <zorder>edtPath</zorder>
 </widget>
 <resources>
  <include location="maya.qrc"/>
 </resources>
 <connections/>
</ui>
